deployment/helpers/wallets.js

```diff
@@ -0,0 +1,39 @@
+const ethers = require("ethers");
+const fs = require('fs');
+
+async function main() {
+  const password = process.argv[2];
```

Comment:
const [firstArg, password] = process.argv

Reply:
the first arg is the script name iiuc which we don't need it